locales/locales.php

```diff
@@ -972,6 +1010,8 @@ public function __construct() {
 		$gsw->country_code = 'ch';
 		$gsw->wp_locale = 'gsw';
 		$gsw->slug = 'gsw';
+		$gsw->variant_root = $de->slug;
```

Comment:
Please note that while German (Switzerland) as a variant of standard German makes sense, Swiss German (gsw) is something completely different. It's a dialect (well, more than 26 different dialects to be precise) that has nothing in common with regular German. As such, `$gsw` shouldn't be a variant of `$de`.